Makefile

```diff
@@ -83,3 +83,4 @@ undeploy:
 gen-manifests:
 	@curl -L $(TOPOLOGYAPI_MANIFESTS)/crd.yaml
 	@hack/get-manifest-ds.sh
+
```

Comment:
next time I'd create a `REBUILD.changelog` top level file which we can use to trigger builds.

Reply:
Every day we learn something new :)

Reply:
Sure, just please note this is only a convention of ours. Such a file (neither the name or the content) has no special meaning for git, github, the CI or anything.

Reply:
Yes, I understood that. It's like ```README``` file convention.
Anyway, I took the opportunity of this PR to implement that.